### DISCUSSION TITLE
Image Library with Modern PnP Search

### DISCUSSION THREAD

**Discussion by Lamvino** [microsoft-search/pnp-modern-search#2782] [Q&A] [answered]
Hello, any help welcome here. We've been using the PnP Modern Search for an Image Library, which works great with the managed metadata we have configured to it. However when the image is selected, it opens up the "Image Preview" page as opposed to the more detailed Image toolbar you receive when opening from the Library itself. Is this a relatively trivial update within the configuration? A couple of sample screenshots below to hopefully help with what I am trying to ask:

![sample1](https://user-images.githubusercontent.com/30998606/219649550-f7b5893d-d5c3-4d03-83c2-ba00cc8d7351.png)
![sample2](https://user-images.githubusercontent.com/30998606/219649567-a1b362b5-52d9-4349-9799-dbcb205ee518.png)

**Comment by wobba** [comment, marked as answer]
I've been meaning to change this but haven't gotten around to it. Point he URL to `<SPWebUrl>/_layouts/15/viewer.aspx?sourcedoc={<uniqueid>}`. Same as Microsoft use in the OOB experiences.

**Reply by Lamvino** [reply]
Thanks a lot for the quick reply! We'll give this a go. Great Search seminar over at ESPC Copenhagen in December by the way 👍
